modules/node_modules/@ncigdc/dux/cart.js

```diff
@@ -219,6 +225,10 @@ function fetchFilesAndRemove(currentFilters: ?Object, size: number): Function {
       return;
     }
 
+    messageNotificationDispatcher('info',
+      <span>Removing <b>{existingFiles.length}</b> files from cart</span>,
```

Comment:
The idea here is to initially show that we are removing whatever number of files we have in cart then go ahead fetch the queried files and only remove those. 
Similar to what we have for adding files in PROD. we will show that we are adding all of the files user currently seeing but then go ahead add only those ones that are not already in the cart. 
What does `size` represent here?